SPECS/redis/redis.spec

```diff
@@ -25,6 +25,7 @@ Redis is an in-memory data structure store, used as database, cache and message
 %prep
 %setup -q
 %patch0 -p1
+%patch1 -p1
```

Comment:
Instead of adding another %patch statement you should switch "%setup -q" to "￼￼%autosetup￼ -p1" and remove the existing %patch statement

Reply:
Fixed